solr/solr-ref-guide/src/configsets-api.adoc

```diff
@@ -89,7 +89,7 @@ A configset is uploaded in a "trusted" mode if authentication is enabled and the
 * If specified in the configset, the DataImportHandler's ScriptTransformer will not initialize.
 * The XSLT transformer (`tr` parameter) cannot be used at request processing time.
 * If specified in the configset, the StatelessScriptUpdateProcessor will not initialize.
-* Collections won't initialize if <lib> directives are used in the configset. If you need to include libraries, please add them to the Solr's classpath.
+* Collections won't initialize if <lib> directives are used in the configset. (Note: Libraries added to Solr's classpath don't need the <lib> directive)
```

Comment:
Could we add a link here to the new docs that @dsmiley is about to add? The one where it lists the places you can put libs.

Reply:
Good idea, Jan. @dsmiley Please "do the needful" ;-)
I'm going to merge this to master and branch_8x now.